three.js/examples/nft.html

```diff
@@ -6,7 +6,7 @@
 <!-- three.js load GLTF -->
 <script src='vendor/three.js/GLTFLoader.js'></script>
 <!-- ar.js -->
-<script src='../build/ar-nft.min.js'></script>
+<script src='../build/ar-nft.js'></script>
```

Comment:
i need to use the not minified lib because an issue with uglifier. The minified version don't load the blob worker...

Reply:
this is not yet solved!

Reply:
We removed the minified version because was only a 100kb difference in size. We will see in the future if it make sense a minified version.